Bottom: 50581eac42ebf825d2b40044c5bd4f51aea3d5f1
Top:    0132521337e539822377263514ee0cea100df275
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-09-04 16:45:16 +0800

Refresh of vhost-split-out-ring-head

---

diff --git a/drivers/vhost/vhost.h b/drivers/vhost/vhost.h
index f41ad64..99ad23a 100644
--- a/drivers/vhost/vhost.h
+++ b/drivers/vhost/vhost.h
@@ -192,10 +192,10 @@ int vhost_get_vq_desc(struct vhost_virtqueue *,
 		      unsigned int *out_num, unsigned int *in_num,
 		      struct vhost_log *log, unsigned int *log_num);
 int __vhost_get_vq_desc(struct vhost_virtqueue *,
-                        struct iovec iov[], unsigned int iov_count,
-                        unsigned int *out_num, unsigned int *in_num,
-                        struct vhost_log *log, unsigned int *log_num,
-                        __virtio16 ring_head);
+			struct iovec iov[], unsigned int iov_count,
+			unsigned int *out_num, unsigned int *in_num,
+			struct vhost_log *log, unsigned int *log_num,
+			__virtio16 ring_head);
 void vhost_discard_vq_desc(struct vhost_virtqueue *, int n);
 
 int vhost_vq_init_access(struct vhost_virtqueue *);
